execution.py

```diff
@@ -94,7 +95,13 @@ def execute(self, execution_context, results):
                 self._logger.warn('Error executing %s:', instruction, 
                                   exc_info=1)
                 stop_test = 'stoptest' in type(error).__name__.lower()
-                results.failed(instruction, error.args[0], stop_test)
+
+                if len(error.args) > 0:
+                    cause = error.args[0]
+                else:
+                    cause = '\n'.join([str(type(error)),traceback.format_exc()])
+
+                results.failed(instruction, cause, stop_test)
```

Comment:
It shows exception as in a normal case for exception and continues test running. Without fix tests are stopped if someone make something like **'raise Exception()'**